src/components/text-area/_text-area.scss

```diff
@@ -109,3 +109,81 @@
     }
   }
 }
+
+@mixin text-area--experimental {
+  .#{$prefix}--text-area {
+    @include reset;
+    @include typescale('zeta');
+    @include focus-outline('reset');
+    background-color: $field-01;
+    width: 100%;
+    min-width: 10rem;
+    padding: $spacing-md;
+    color: $text-01;
+    order: 2;
+    resize: vertical;
+    border: none;
+    border-bottom: 1px solid $ui-04;
+    transition: $transition--base all;
+  }
+
+  .#{$prefix}--text-area:hover {
+    background-color: $hover-field;
+  }
+
+  .#{$prefix}--text-area:focus,
+  .#{$prefix}--text-area:active {
+    @include focus-outline('outline');
+  }
+
+  .#{$prefix}--text-area::-webkit-input-placeholder {
+    @include placeholder-colors;
+  }
+
+  //-----------------------------
+  // Disabled & Error icon spacing
+  //-----------------------------
+  .#{$prefix}--text-area:disabled,
+  .#{$prefix}--text-area[data-invalid],
+  .#{$prefix}--text-area:invalid,
+  .#{$prefix}--text-area[data-loading] {
+    padding-right: rem(40px);
+    background-repeat: no-repeat;
+    background-position: right 8px bottom 50%;
```

Comment:
More of a question for @IBM/carbon-designers (unless you've already asked), is the error icon supposed to be centered or aligned to the top right / bottom right? 

![screen shot 2018-11-05 at 3 08 55 pm](https://user-images.githubusercontent.com/11928039/48026850-d6899c80-e10c-11e8-926d-3dde3eb64e73.png)

Reply:
Totally good question! Wasn't sure about that, but can definitely change to whatever it needs to be.

Reply:
Top-aligned, vertical center with the text just like notifications is the rule for icons... (although we're doing an audit of these error states and error icons this week)